.idea/.idea.ProjectLighthouse/.idea/dataSources.xml

```diff
@@ -4,7 +4,7 @@
     <data-source source="LOCAL" name="lighthouse@localhost" uuid="3bfb85d6-6cc2-427d-9288-06e05ab58c10">
       <driver-ref>mysql.8</driver-ref>
       <synchronize>true</synchronize>
-      <jdbc-driver>com.mysql.cj.jdbc.NonRegisteringDriver</jdbc-driver>
```

Comment:
Why was this file ever commited, it's one of those that should be in the gitignore

Reply:
No idea, that's a Slendy question